canonical/cloud-init
# Fix not sourcing default 50-cloud-init file

## Proposed Commit Message
Fix not sourcing default 50-cloud-init file

## Additional Context
While provisioning under debian, a line is added to ENI sourcing *.cfg.
However, the default file written by cloud-init is 50-cloud-init (so not 50-cloud-init.cfg, thats only for Ubuntu)

This fixes the incorrect filename being sourced
## Test Steps
1- Boot debian 10.6.0 standard install with VMware postconfiguration
2- Check contents of ENI file, it will show "source /etc/networks/interfaces.d/50-cloud-init.cfg", where VMware postconfig generates a "50-cloud-init" (note the missing suffix) file.

## Checklist:
 - [X] My code follows the process laid out in [the documentation]
